app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php

```diff
@@ -190,10 +190,34 @@ public function isSupportedProductType()
     protected function getProductSupportedTypes()
     {
         return [
-            Mage_Catalog_Model_Product_Type::TYPE_SIMPLE
+            Mage_Catalog_Model_Product_Type::TYPE_SIMPLE,
+            Mage_Catalog_Model_Product_Type::TYPE_CONFIGURABLE,
         ];
     }
 
+    /**
+     * Get maximum available quantity of product
+     * It is used to show a warning if user tries to buy more then product quantity
+     * @return int -1 Ignore product quantity levels
+     *              0 if product is out of stock
+     *              positive value otherwise
+     *
+     * @deprecated
+     */
+    public function getProductMaxQty()
```

Comment:
Why was this function added back? I don't think it's used anywhere.

Reply:
The result of merging with conflicts.  It's a miss on my part.  I'm removing it.

Reply:
There were actually no unit tests for this before.

Reply:
ok. there was a bad merge on my end after trying to resolve conflicts.  The method has been removed.